x-pack/legacy/plugins/file_upload/public/components/json_index_file_picker.js

```diff
@@ -135,75 +130,55 @@ export class JsonIndexFilePicker extends Component {
       }
       setFileRef(file);
       setParsedFile(parsedFileResult);
-
     } else {
       // No else
     }
-  }
+  };
 
   render() {
     const { fileParsingProgress, fileUploadError, fileRef } = this.state;
 
     return (
       <Fragment>
-        { fileParsingProgress
-          ? <EuiProgress size="xs" color="accent" position="absolute" />
-          : null
-        }
-        {
-          fileRef && !fileUploadError
-            ? null
-            : (
-              <EuiCallOut
-                title={i18n.translate(
-                  'xpack.fileUpload.jsonIndexFilePicker.fileUploadGuidelines',
-                  { defaultMessage: 'File upload guidelines' }
-                )}
-                iconType="pin"
-              >
-                <div>
-                  <ul>
-                    <li>
-                      {
-                        i18n.translate(
-                          'xpack.fileUpload.jsonIndexFilePicker.formatsAccepted',
-                          { defaultMessage: 'Formats accepted: .json, .geojson' }
-                        )
-                      }
-                    </li>
-                    <li>
-                      <FormattedMessage
-                        id="xpack.fileUpload.jsonIndexFilePicker.maxSize"
-                        defaultMessage="Max size: {maxFileSize}"
-                        values={{
-                          maxFileSize: bytesToSize(MAX_FILE_SIZE)
-                        }}
-                      />
-                    </li>
-                  </ul>
-                </div>
-              </EuiCallOut>
-            )
-        }
-        <EuiSpacer size="m" />
+        {fileParsingProgress ? <EuiProgress size="xs" color="accent" position="absolute" /> : null}
+        {fileRef && !fileUploadError ? null : null}
```

Comment:
@aaronjcaldwell AFAICT, This `fileRef` was only used as a check on whether to display the file upload guidelines. But since I moved them to the help text, it should always be there. I couldn't get rid of it completely. So let me know how you'd like me to handle this variable. Or, you're also welcome to push commits directly to my branch for any fixes you find.

Reply:
You should be able to remove line 143 as well as its retrieval from state on 138. The other uses in this component relate to removing the file and should be left in for now.